MdeModulePkg/Bus/Pci/PciBusDxe/PciResourceSupport.c

```diff
@@ -1828,11 +1828,13 @@ ResourcePaddingForCardBusBridge (
            PciBarTypeMem32,
            PciResUsagePadding
            );
-
-  InsertResourceNode (
-    Mem32Node,
-    Node
-    );
+  // MU_CHANGE - Handle CreateResourceNode returning a NULL
+  if (Node != NULL) {

```

Comment:
This is creating a resouce padding for a cardbus bridge, so it can handle hot plugging.

The function is creating padding for Memory, Prefetchable Memory, and I/O. if a specific padding is not available, I think the code should still enable padding for any of the other resources. That way a device that would only need Memory would still work even if a device that needed I/O could fail.

Reply:
I think this would be most compatible with previous behavior which would continue on (and cause a mess inside `InsertResourceNode()`).

Given that alrady asserts if an incoming argument is NULL, I think that's sufficient to let someone know something unexpected happened in a DEBUG build.